pkg/windows/calico.go

```diff
@@ -283,7 +283,7 @@ func (c *Calico) Start(ctx context.Context) error {
 // generateCalicoNetworks creates the overlay networks for internode networking
 func (c *Calico) generateCalicoNetworks() error {
 	if err := deleteAllNetworks(); err != nil {
-		return err
+		return fmt.Errorf("error deleting all networks before bootstrapping calico: %v", err)
```

Comment:
For wrapping errors, the convention in this file seems to be:
```suggestion
		return errors.Wrap(err, "failed to delete all networks before bootstrapping calico")
```

Reply:
There are both options present in this file... let me change all to errors.Wrap